replica/table.cc

```diff
@@ -713,6 +709,10 @@ class table_sstables_adder : public row_cache::external_updater_impl {
             tlogger.warn("table_sstables_adder: some sstables are already in the table: candidates={} inserted={}", _sstables, inserted);
             _sstables = std::move(inserted);
         }
+        if (_is_main) {
+            _new_backlog_tracker.emplace(_cg.get_backlog_tracker().clone());
+            _new_backlog_tracker->replace_sstables({}, _sstables);
+        }
```

Comment:
as we discussed, this is doing cloning work twice, as replace_sstables also clones the tracker state in order to provide strong exception guarantees. let's avoid redundant work by unifying clone and replace_sstables (the former is already done today anyway)

Reply:
next version will introduce clone_and_replace_sstables that will combine the exception-safe replace_sstables with cloning the tracker contents.